src/components/pages/Analytics/AnalyticsChartBox.tsx

```diff
@@ -1,15 +1,11 @@
 import { Box } from '@chakra-ui/react';
 import useSize from '@react-hook/size';
 import { EChartsInstance } from 'echarts-for-react';
-import dynamic from 'next/dynamic';
 import React, { useLayoutEffect, useRef } from 'react';
 
 import { BorderedCard } from '~/components/base';
 import ChartHeader from '~/components/shared/Charts/ChartHeader';
-const HistoricalChart = dynamic(
-  () => import('~/components/shared/Charts/HistoricalChart'),
-  { ssr: false },
-);
+import HistoricalChart from '~/components/shared/Charts/HistoricalChart';
 import { useSingleLineChart, UseSingleLineChartProps } from '~/hooks/useChart';
```

Comment:
@v1nvn 👀  I have noticed the transpiler package in place. Did you face challenge with dynamic loading option 😄 🤔 ?
I am curios.

Reply:
When importing echarts using `import * as echarts from 'echarts/core'` in Historical/Bar Charts, it throws an exception when building:

```
export * from './lib/export/charts';
^^^^^^

SyntaxError: Unexpected token 'export'
```

Apparently you cannot import echarts/core or anything from lib directory inside the app since it runs in a node environment and the code is transpiled to ES2015 (it needs CommonJS). So this plugin transpiles it down to CommonJS.

Reply:
> As for the dynamic imports, I removed them because it required setting static height manually for chart's container to prevent layout shift while it loads. Since we are already on React 18, recommended way would be to wrap dynamically imported components in `Suspense` with a loading placeholder which compensates for layout shift without setting width/height manually. That's a separate task altogether, we'll tackle that later on in a separate ticket.

ah yes, there was the cls effect that needed a static height to be set, suspense will be great addition.

Reply:
> Unexpected token 'export'

@v1nvn interesting. Yes I remember this error, but what I could only see from the community is that it seems to be as a result of SSR https://github.com/hustcc/echarts-for-react/issues/425 so if you disable ssr during import, then it works.

also this reply https://github.com/apache/echarts/issues/14864#issuecomment-859634483